sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -192,7 +181,7 @@ namespace Azure { namespace Storage { namespace Blobs {
      * parameters to execute this function.
      * @return A SetAccessTierResponse on successfully setting the tier.
      */
-    Azure::Core::Response<SetAccessTierResponse> SetAccessTier(
+    Azure::Core::Response<SetAccessTierInfo> SetAccessTier(
```

Comment:
Can we be absolutely sure that there won't be any container access tier in the future? If not, then there would be breaking change or `SetAccessTierInfo` and `SetContainerAccessTierInfo`.

Reply:
Why would a container have access tier? It doesn't make any sense.

Reply:
There might be the case that customer wants to manage the access tier at the container level. I'm not sure if that's going to be added in the future.
What I want to point out is that `SetAccessTierInfo` does not limit the scope to be `Blob`, then any addition to the scope under the `Azure::Storage::Blobs` namespace would be breaking change in the future. If this is considered not possible in your mind, I'm fine with the current naming.

Reply:
fixed